content/getting-started/livechat-apps/webhook-apps/index.mdx

```diff
@@ -9,102 +9,106 @@ desc: "Learn how to build your own integration using LiveChat webhooks."
 
 ## 5. Create a settings page
 
-Settings page is the place where you can:
+Settings page is a place where you can:
 
-- ask users to connect their accounts with your OAuth2.0 provider,
+- prompt users to connect their accounts with your OAuth2.0 provider,
 - display configuration options,
 - inform users about additional installation steps (if there are any).
 
-If you need to access LiveChat user data, we recommend using [Sign in with LiveChat SDK](/getting-started/authorization/sign-in-with-livechat/).
+If you need to access LiveChat user data, we recommend using [Sign in with LiveChat authorization flow](/getting-started/authorization/sign-in-with-livechat/). For more information on possible widget placements, visit the [app locations](/extending-agent-app/#app-locations) page.
 
 ### Sample settings page scenario
 
-For instance if you're building an integration that binds LiveChat data with an external service, you should create a settings page that:
+If you're building an integration that binds LiveChat data with an external service, you should create a settings page that:
 
-1. Imports [Sign in with LiveChat SDK](/getting-started/authorization/sign-in-with-livechat/).
+1. Imports [Sign in with LiveChat flow](/getting-started/authorization/sign-in-with-livechat/).
 2. Imports your sign in form or other authorization flow (e.g. "Connect with..." button).
-3. Binds both informations and stores new user in your database.
+3. Binds both pieces of information and stores the new user in your database.
 4. Displays a confirmation screen when the integration is up and running.
 
-The settings page will appear right after LiveChat user installs your app in the Marketplace.
+The settings page will appear right after LiveChat user installs your app from the Marketplace.
 
 ## 6. Register the webhooks
 
-Once you have LiveChat credentials, you can register your webhooks with Platform APIs. There are two APIs you can use to manage webhooks:
+Once you have the LiveChat credentials, you can register your webhooks with the Configuration API. Currently, there are two APIs you can use to manage webhooks:
 
 - [Configuration API v3](/management/configuration-api/#webhooks)
 - [Configuration API v2](/management/configuration-api/v2.0/#create-a-new-webhook) (deprecated)
 
 ### Seting up webhooks for development
 
-For development purposes we recommend setting your webhooks from within LiveChat Agent App.
+One of the possible development options is to set up your webhooks from within the LiveChat Agent App. However, please note that this configuration uses [Configuration API v2 webhooks](/management/configuration-api/v2.0/#create-a-new-webhook), which are now deprecated.
+
+We highly recommend using the [Configuration API v3 webhooks](/management/configuration-api/#webhooks). To view a tutorial on setting these up, follow [this link](/getting-started/app-guides/chat-webhooks/#tutorial).
```

Comment:
```suggestion
We highly recommend using the [Configuration API v3 webhooks](/management/configuration-api/#webhooks).

Reply:
It's better not to link that at the moment. v3.3 introduces important changes that we'll need to include in this docs